Source/cmake/FindGIDocgen.cmake

```diff
@@ -60,6 +60,7 @@ if (NOT ENABLE_DOCUMENTATION)
 endif ()
 
 find_package(GI REQUIRED)
+find_package(Unifdef REQUIRED)
```

Comment:
Please make sure this still works when building with `-DUSE_SYSTEM_UNIFDEF=OFF`?

Reply:
Good point, I'll make sure before landing.

Reply:
So, this line is not needed, because we already have this in `Source/CMakeLists.txt`:

```cmake
if (USE_SYSTEM_UNIFDEF)
  find_package(Unifdef REQUIRED)
else ()
  add_subdirectory(ThirdParty/unifdef)
endif ()
```

So I'll remove it before landing.

(Now, a different discussion is whether we should fix the above to avoid checking for `unifdef` or building it for ports other than WPE/GTK. That would belong in another patch.)